play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -813,8 +876,18 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 			</ChatLog>
 			<ChatTextEntry room={room} onMessage={this.send} onKey={this.onKey} left={640} />
 			<ChatUserList room={room} left={640} minimized />
+			<button
+				class="icon button"
+				name="openBattleOptions"
```

Comment:
```suggestion
```

You don't need `name` when you're using `data-href` or `data-cmd`.

We do still use `name`/`value` for the format/team dropdowns, but they're not used for stuff like this.